script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt

```diff
@@ -350,7 +350,7 @@ Apparently, Senpai is not in good spirits.
 "T-That's not true. I'm sorry for asking you such a strange question."
 <Page110>
 Senpai forces herself to act cheerful.
-This topic might have been fine for a long-time friend like Arihiko but for someone like Senpai, such a complex topic must've been bothersome at best.
+This topic might have been fine for a long-time friend like Arihiko but for someone like Senpai, such a complex topic must have been bothersome at best.
```

Comment:
must've

Reply:
I think I read "must have" as "mustuv" but I still expect it to be written "must have".

Reply:
onore americans